utils.c

```diff
@@ -69,6 +69,32 @@ gboolean janus_strcmp_const_time(const void *str1, const void *str2) {
 	return result == 0;
 }
 
+guint32 janus_random_uint32(void) {
+	return g_random_int();
+}
+
+guint64 janus_random_uint64(void) {
+	/*
+	 * FIXME This needs to be improved, and use something that generates
+	 * more strongly random stuff... using /dev/urandom is probably not
+	 * a good idea, as we don't want to make it harder to cross compile Janus
+	 *
+	 * TODO Look into what libssl and/or libcrypto provide in that respect
+	 *
+	 * PS: JavaScript only supports integer up to 2^53, so we need to
+	 * make sure the number is below 9007199254740992 for safety
+	 */
+	guint64 num = g_random_int() & 0x1FFFFF;
+	num = (num << 32) | g_random_int();
```

Comment:
Yeah it's more of a bitwise operation here, although a plus would get you the same result.